build-tools/packages/build-tools/src/fluidBuild/buildGraph.ts

```diff
@@ -221,12 +223,14 @@ export class BuildPackage {
 
 		const task = TaskFactory.Create(this, command, this.context, pendingInitDep, taskName);
 		pendingInitDep.push(task);
+		this.tasks.push(task);
+		this.scriptTasks.set(taskName, task);
 		return task;
 	}
 
 	// Create or return and existing task with a name.  If it is a script, it will also create an return the pre/post script task if it exists
```

Comment:
nit

```suggestion
	// Create or return and existing task with a name.  If it is a script, it will also create and return the pre/post script task if it exists
```